### PR TITLE
Chore/315 token dapp misc fixes

### PR DESCRIPTION
**Pull request by mattrussell36**
# Related issues 🔗

#315 

# Description ℹ️

- Adds a currency lozenge to token input
- Adds loader components to callouts for pending transaction states
- Fixes font colors and sizes of various titles
- Improves spacing in various places
- Aligns header with explorer, removed fish icon for fairground
- Removes unused details prop from Lozenge
- Fixes title spacing on callouts
- Deletes loader component local to token app

# Demo 📺

Screen Shot 2022-05-26 at 6 08 13 PM

### CODE REVIEW

**Review by dexturr — APPROVED** [review]

File: apps/token/src/components/eth-wallet/eth-wallet.tsx [hunk 1]
```diff
@@ -163,15 +163,13 @@ const ConnectedKey = () => {
...
-          <Button variant="primary" className="w-full">
-            {t('associate')}
-          </Button>
+        <Link className="flex-1" to={`${Routes.STAKING}/associate`}>
+          <button className="w-full px-28 border h-28">{t('associate')}</button>
         </Link>
-        <Link style={{ flex: 1 }} to={`${Routes.STAKING}/disassociate`}>
-          <Button variant="primary" className="w-full">
+        <Link className="flex-1" to={`${Routes.STAKING}/disassociate`}>
+          <button className="w-full px-28 border h-28">
```
Comment (notbot00) [inline apps/token/src/components/eth-wallet/eth-wallet.tsx:170]: i think buttons are not valid when nested under anchor elements. 

quote: "The [a](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element) element can be wrapped around entire paragraphs, lists, tables, and so forth, even entire sections, so long as there is no interactive content within (e.g., buttons or other links)" (ref: https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element)

File: apps/token/src/routes/staking/staking.tsx [hunk 2]
```diff
@@ -30,20 +30,22 @@ export const Staking = ({ data }: { data?: StakingQueryResult }) => {
         <p className="mb-12">{t('stakingDescription3')}</p>
         <p className="mb-12">{t('stakingDescription4')}</p>
         <p className="mb-12">
-          <Link href={Links.STAKING_GUIDE} target="_blank">
+          <Link
+            href={Links.STAKING_GUIDE}
+            className="text-white underline"
```
Comment (notbot00) [inline apps/token/src/routes/staking/staking.tsx:35]:
```suggestion
            className="dak:text-white underline"
```

File: apps/token/src/routes/withdraw/index.tsx [hunk 3]
```diff
@@ -140,9 +140,9 @@ export const WithdrawContainer = ({ currVegaKey }: WithdrawContainerProps) => {
             title={t('pendingWithdrawalsCalloutTitle')}
             intent={Intent.Prompt}
           >
-            <p>{t('pendingWithdrawalsCalloutText')}</p>
-            <p className="mb-0">
-              <Link to={Routes.WITHDRAWALS}>
+            <p className="mb-8">{t('pendingWithdrawalsCalloutText')}</p>
+            <p>
+              <Link to={Routes.WITHDRAWALS} className="underline text-white">
```
Comment (notbot00) [inline apps/token/src/routes/withdraw/index.tsx:145]:
```suggestion
              <Link to={Routes.WITHDRAWALS} className="underline dark:text-white">
```

File: apps/token/src/components/app-footer/app-footer.tsx [hunk 4]
```diff
@@ -13,8 +14,12 @@ export const AppFooter = () => {
           i18nKey="footerLinksText"
           components={{
             /* eslint-disable */
-            feedbackLink: <a href={Links.FEEDBACK} />,
-            githubLink: <a href={Links.GITHUB} />,
+            feedbackLink: (
+              <Link className="text-white underline" href={Links.FEEDBACK} />
```
Comment (notbot00) [inline apps/token/src/components/app-footer/app-footer.tsx:18]:
```suggestion
              <Link className="dark:text-white underline" href={Links.FEEDBACK} />
```

**Review by notbot00 — APPROVED** [review]

File: apps/token/src/routes/staking/node-list.tsx [hunk 5]
```diff
@@ -46,7 +46,7 @@ export const NODES_QUERY = gql`
 `;
 
 const NodeListItemName = ({ children }: { children: React.ReactNode }) => (
-  <span className="mr-4 underline">{children}</span>
+  <span className="mr-4 underline text-white">{children}</span>
```
[the remark this answers is not in the document]
Comment (mattrussell36) [reply apps/token/src/routes/staking/node-list.tsx:49]: Yup no theme on this one 👍

**Pull request merged**